src/status_im/contexts/profile/contact/share/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im.contexts.profile.contact.share.view
+  (:require [legacy.status-im.ui.components.list-selection :as list-selection]
+            [quo.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.qr-codes.view :as qr-codes]
+            [status-im.contexts.profile.contact.share.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.address :as address]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [universal-profile-url customization-color]
+         :as   profile} (rf/sub [:contacts/current-contact])
+        abbreviated-url (rn/use-memo (fn []
+                                       (address/get-abbreviated-profile-url universal-profile-url))
+                                     [universal-profile-url])
+        profile-picture (rn/use-memo #(profile.utils/photo profile) [])
+        display-name    (rn/use-memo #(profile.utils/displayed-name profile) [])
+        on-back-press   (rn/use-callback #(rf/dispatch [:navigate-back]))
+        on-share-press  (rn/use-callback #(list-selection/open-share {:message universal-profile-url}))
```

Comment:
is this intentional, seems like `universal-profile-url` should also be in the deps for this use of use-callback